weasyl/templates/help/gdocs.html

```diff
@@ -1,50 +1,45 @@
-$:{TITLE("Google Drive Embedding", "Help", "/help")}
+$:{TITLE("Google Docs Embedding", "Help", "/help")}
 
 <div class="content">
   <div class="constrained">
     <h3>How do I find the embed link?</h3>
 
-    <h4>For documents</h4>
-
-    <p>
-      In the <em>File</em> menu of your document, there is a <em>Publish to the web…</em> option.
-    </p>
-
-    <figure class="help-image">
-      <img src="${resource_path('img/help/help-gdocs-1.png')}" width="276" height="334">
-    </figure>
-
-    <p>
-      In this section, select <em>Start publishing</em>.
-    </p>
-
-    <figure class="help-image">
-      <img src="${resource_path('img/help/help-gdocs-2.png')}" width="346" height="150">
-    </figure>
-
-    <p>
-      The embed link is located inside the <em>Embed code</em> block of
-      HTML. Pasting just the link is not necessary; Weasyl will take the link
-      out of the whole embed code.
-    </p>
-
-    <figure class="help-image">
-      <img src="${resource_path('img/help/help-gdocs-3.png')}" width="516" height="182">
-    </figure>
-
-    <h4>For anything else</h4>
-
-    <p>
-      Instead of a <em>Publish to the web…</em> option, PDFs will have
-      an <em>Embed this PDF file…</em> option. Anything else will have
-      an <em>Embed link…</em> option. The block of HTML in the
-      resulting dialog contains the embed link. As with documents, pasting just
-      the link is not necessary; Weasyl will take the link out of the whole
-      embed code.
-    </p>
-
-    <figure class="help-image">
-      <img src="${resource_path('img/help/help-gdocs-4.png')}" width="540" height="88">
-    </figure>
+    <ol>
+      <li>
+        <figure class="help-image">
+          <figcaption>In the <kbd><samp>File</samp></kbd> menu of your document, there is a <kbd><samp>Publish to the web</samp></kbd> option.</figcaption>
```

Comment:
Question: Why ``<kbd><samp>...</samp></kbd>``? From a visual standpoint--currently, at least--it _feels_ like a more verbose ``<em>...</em>``/bolding; according to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd), the `<kbd>` element ``represents a span of inline text denoting textual user input from a keyboard, voice input, or any other text entry device. By convention, the user agent defaults to rendering the contents of a <kbd> element using its default monospace font, although this is not mandated by the HTML standard.``

Looking into ``<samp>`` as well, it ``is used to enclose inline text which represents sample (or quoted) output from a computer program``.

And I do see that the ``help.scss`` does set style on `kbd`/`samp`, but does this completely override User-Agent applied styles? (I'm mostly just theorizing at this point... from a usability perspective, it _shouldn't_ affect it, but it's something I noticed, was curious about, and thus I just figured I'd make a note.)

Reply:
Lower on the page: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd#representing_onscreen_input_options

It’s a bit weird, but it’s specifically in [the HTML spec](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-kbd-element) too:

> When the `kbd` element contains a `samp` element, it represents input based on system output, for example invoking a menu item.

Supposedly just `kbd` is fine for the same purpose, but I figured I may as well. :)